init-db.sql

```diff
@@ -4,5 +4,93 @@ CREATE TABLE `internet_shop`.`products` (
   `product_id` BIGINT(11) NOT NULL AUTO_INCREMENT,
   `name` VARCHAR(255) NOT NULL,
   `price` DOUBLE NOT NULL,
- 'deleted' BOOLEAN NOT NULL DEFAULT false,
+ `deleted` BOOLEAN NOT NULL DEFAULT false,
  PRIMARY KEY (`product_id`));
+
+ CREATE TABLE `internet_shop`.`users` (
+  `user_id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(255) NOT NULL,
+   `login` VARCHAR(255) NOT NULL,
+    `password` VARCHAR(255) NOT NULL,
+ `deleted` BOOLEAN NOT NULL DEFAULT false,
+ PRIMARY KEY (`user_id`));
+
+ CREATE TABLE `internet_shop`.`roles` (
+  `role_id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+  `role_name` VARCHAR(255) NOT NULL,
+ `deleted` BOOLEAN NOT NULL DEFAULT false,
+ PRIMARY KEY (`role_id`));
+
+ INSERT INTO `roles`(role_name)
+VALUES ('ADMIN');
+INSERT INTO `roles`(role_name)
+VALUES ('USER');
+
+ CREATE TABLE `internet_shop`.`shopping_carts` (
+  `cart_id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+  `user_id` BIGINT(11) NOT NULL,
+ `deleted` BOOLEAN NOT NULL DEFAULT false,
+ PRIMARY KEY (`cart_id`),
+ CONSTRAINT fk_userIDtoSPcart
+    FOREIGN KEY (`user_id`)
+    REFERENCES internet_shop.users(`user_id`)
+        ON UPDATE RESTART
```

Comment:
Just a question out of curiosity)) Why do you want to restart MySQL on update/delete in shopping_carts? RESTART restarts MySQL server, right?